src/pages/git/git-checkout/index.md

```diff
@@ -3,13 +3,38 @@ title: Git Checkout
 ---
 ## Git Checkout
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-checkout/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Git Checkout switches between branches or restore working tree files.  There are a lot of options to checkout that won't be covered here but you can take a look at all of them [here][checkout-options] 
```

Comment:
Minor nit: "restore" -> "restores"

For the link text, instead of using "here", let's say "in the Git documentation."